src/analysis/import/import_neighborhood.sh

```diff
@@ -76,58 +83,79 @@ then
+                -c "
+            CREATE TABLE IF NOT EXISTS \"water_blocks\" (
+                \"STATEFP10\" integer,
+                \"COUNTYFP10\" integer,
+                \"TRACTCE10\" integer,
+                \"BLOCKCE10\" integer,
+                GEOID varchar(15),
+                \"NAME10\" char(10),
+                \"MTFCC10\" char(5),
+                \"UR10\" char(1),
+                \"UACE10\" integer,
+                \"UATYP10\" char(1),
+                \"FUNCSTAT10\" char(1),
+                \"ALAND10\" integer,
+                \"AWATER10\" bigint,
+                \"INTPTLAT10\" decimal,
+                \"INTPTLON10\" decimal
+            );"
+
+            # Import water file
+            # Only if in USA
+            WATER_FILENAME="censuswaterblocks"
+            WATER_DOWNLOAD="${NB_TEMPDIR}/${WATER_FILENAME}.zip"
+            wget -nv -O "${WATER_DOWNLOAD}" "https://s3.amazonaws.com/pfb-public-documents/censuswaterblocks.zip"
+            unzip "${WATER_DOWNLOAD}" -d "${NB_TEMPDIR}"
+            psql -h "${NB_POSTGRESQL_HOST}" -U "${NB_POSTGRESQL_USER}" -d "${NB_POSTGRESQL_DB}" \
+                -c "\copy water_blocks FROM ${NB_TEMPDIR}/${WATER_FILENAME}.csv delimiter ',' csv header"
+            echo "DONE: Importing water blocks"
+        fi
 
-        # Get blocks for the state requested
+        # Get blocks for the place requested
         update_status "IMPORTING" "Downloading census blocks"
-        NB_BLOCK_FILENAME="tabblock2010_${NB_STATE_FIPS}_pophu"
+        if [[ -n "${NB_STATE_FIPS}" ]]; then
+            NB_BLOCK_FILENAME="tabblock2010_${NB_STATE_FIPS}_pophu"
+        else
+            NB_BLOCK_FILENAME="population"
+        fi
+
         S3_PATH="s3://${AWS_STORAGE_BUCKET_NAME}/data/${NB_BLOCK_FILENAME}.zip"
         if [ -f "/data/${NB_BLOCK_FILENAME}.zip" ]; then
             echo "Using local census blocks file"
             BLOCK_DOWNLOAD="/data/${NB_BLOCK_FILENAME}.zip"
+        elif [ -n "${PFB_POP_URL}" ]; then
+            echo "Using blocks file from PFB_POP_URL: ${PFB_POP_URL}"
+            BLOCK_DOWNLOAD="${NB_TEMPDIR}/${NB_BLOCK_FILENAME}.zip"
+            wget -nv -O "${BLOCK_DOWNLOAD}" "${PFB_POP_URL}"
         elif [ "${AWS_STORAGE_BUCKET_NAME}" ] && aws s3 ls "${S3_PATH}"; then
             echo "Using census blocks file from S3"
             BLOCK_DOWNLOAD="${NB_TEMPDIR}/${NB_BLOCK_FILENAME}.zip"
             aws s3 cp "${S3_PATH}" "${BLOCK_DOWNLOAD}"
         else
+            echo "Using census blocks file from official census site"
             BLOCK_DOWNLOAD="${NB_TEMPDIR}/${NB_BLOCK_FILENAME}.zip"
-            wget -nv -O "${BLOCK_DOWNLOAD}" "http://www2.census.gov/geo/tiger/TIGER2010BLKPOPHU/${NB_BLOCK_FILENAME}.zip"
+            PFB_POP_URL="http://www2.census.gov/geo/tiger/TIGER2010BLKPOPHU/${NB_BLOCK_FILENAME}.zip"
+            wget -nv -O "${BLOCK_DOWNLOAD}" "${PFB_POP_URL}"
```

Comment:
For a US town with no population file URL provided, I'm ending up here but it's doing this:
> Using census blocks file from official census site
https://www2.census.gov/geo/tiger/TIGER2010BLKPOPHU/population.zip:
2022-07-22 13:50:50 ERROR 404: Not Found.

So seemingly `NB_STATE_FIPS` isn't set, but I'm not sure why not.